utils/land-patterns.stanza

```diff
@@ -147,11 +147,13 @@ public defn apply-soldermask (mask-shape:Shape):
 public defn apply-soldermask (amount:Double, expand?:True|False):
   val mask-shape = 
     match(pad-shape(self)):
-      (s:Rectangle|Circle|RoundedRectangle|Segment):
+      (s:Rectangle|Circle|RoundedRectangle|Segment|Capsule):
```

Comment:
`offset` is implemented for `Capsule`, so we can include it here.

Reply:
This behaves differently than before, and a big part of the original discussion was that the solder mask just keep the same shape as the original. So the solder mask for a Rectangle should still be just a simple Rectangle. Is this requirement still true?

Reply:
We're using `offset` here as the default because it is better defined for what "solder mask registration" means (the distance away from the pad/mask shape that defines the pad).